iotdb-core/datanode/src/main/java/org/apache/iotdb/db/tools/validate/TsFileValidationTool.java

```diff
@@ -189,351 +160,10 @@ public static void main(String[] args) throws IOException {
 
   public static void findUncorrectFiles(List<File> tsFiles) {
```

Comment:
Do we need to check for out-of-order files as well? Out-of-order files do not need to check for overlap between files, but a single file may still have errors(chunk level or page level) checked in TsFileValidationScan

Reply:
Added a parameter `ignoreFileOverlap(-if)`. 
When it is true, at the end of the scan of each file, the context maps will be cleared so that overlap between files will not be detected.